src/App.js

```diff
@@ -1,19 +1,22 @@
 import { Component } from 'react';
-import logo from './react-logo.svg';
 import './App.css';
 import React from 'react';
-
+import Header from './Header';
+import Footer from './Footer';
+import HeroList from './HeroList';
+import heroes from './Data';
 class App extends Component {
 
   render() {
-    return (
-      <div className="App">
-  
-        My React App...
-        
-        <img src={logo} className="temp-images" alt="react logo" />
-        <img src="acl-logo.png" className="temp-images" alt="acl logo" />
-      
+    return ( 
+      <div className='App'>
```

Comment:
Nice work importing heros at the top could clean up the white paces a few but overall looks good